### PR TITLE
Link to docs on the Foxglove website

### PR DESCRIPTION
**Pull request by esthersweon**
Per @amacneil 's request.

**Public-Facing Changes**

- Remove docs from README
- Link to docs on the Foxglove website
- Blocked by https://github.com/foxglove/website/pull/971

### CODE REVIEW

**Comment by defunctzombie** [comment]
@amacneil What's the reasoning here? As a developer looking to use a package like this I often find the README a pretty good place for docs. Granted some projects do decide to host docs separately from the README.

**Comment by achim-k** [comment]
> @amacneil What's the reasoning here? As a developer looking to use a package like this I often find the README a pretty good place for docs. Granted some projects do decide to host docs separately from the README.

I agree, I also find the README helpful and I would not remove it. The big advantage is versioning:
Documentation updates is linked to code changes & releases. For instance, https://index.ros.org/p/foxglove_bridge does not show the latest README but only the version of the latest releases. I believe this is also what we should do for the foxglove-bridge docs that are to be hosted on the foxglove website. I see little value in also documenting unreleased foxglove-bridge features on the foxglove website.

File: README.md [hunk 1]
```diff
@@ -15,89 +15,35 @@ Live debugging of ROS systems has traditionally relied on running ROS tooling su
...
-## Clients
-
-[Foxglove Studio](https://foxglove.dev/studio) connects to `foxglove_bridge` for live robotics visualization.
-
-### Building from source
-
-You can also try `foxglove_bridge` by building from source or running a pre-built Docker container.
-
-#### Fetch source and install dependencies
+To build `foxglove_bridge` from source, you must first fetch your source and install dependencies:
```
Comment (defunctzombie) [inline README.md:24]:
```suggestion
To build `foxglove_bridge` from source, you should clone the source to your ROS workspace and install dependencies:
```

**Review by defunctzombie — APPROVED** [review]

**Review by jhurliman — CHANGES_REQUESTED** [review]
~Net negative change~

EDIT: Achim wrote up a pretty good description of why erasing the README for a ROS package is not a good idea. I'm marking this as "changes requested" to hold up the PR for discussion.

**Comment by amacneil** [comment]
Two anti patterns I would like us to avoid:
- Having multiple sets of documentation for the same thing. It's fine for the readme to be a "jumping off point", but it should send you to the canonical documentation.
- Forcing people to jump between multiple docs sites and readme's to solve a problem. For example, before we started cleaning up our readme's, it was necessary to jump between `ws-protocol`, `foxglove/schemas`, and now `ros-foxglove-bridge` to gain all the context on how to connect your robot to studio. I should go to one place for this information and not need to find my way through a web of github repos.

So it's fine to have some basic info in the readme but let's point people to the canonical docs.

**Comment by defunctzombie** [comment]
> Net negative change

@jhurliman can you please be more specific? This type of comment is not helpful since there is no clear action to take in response to your change request.

**Comment by jhurliman** [comment]
Discussed in Slack, we are going to rework the docs approach to keep per-version specific instructions such as launchfile instructions and parameter descriptions in git while maintaining high-level overview and links to instructions on the foxglove.dev/docs site.

**Pull request closed without merging**